modules/nft/keeper/keeper.go

```diff
@@ -117,10 +117,16 @@ func (k Keeper) TransferOwner(
 	ctx sdk.Context, denomID, tokenID, tokenNm, tokenURI,
 	tokenData string, srcOwner, dstOwner sdk.AccAddress,
 ) error {
-	if !k.HasDenomID(ctx, denomID) {
+	denom, found := k.GetDenom(ctx, denomID)
+	if !found {
 		return sdkerrors.Wrapf(types.ErrInvalidDenom, "denom ID %s not exists", denomID)
 	}
 
+	if denom.UpdateRestricted {
```

Comment:
`denom.UpdateRestricted` is true means that this NFT cannot be updated instead of transferring the ownership.